src/neo/Ledger/ContractState.cs

```diff
@@ -13,29 +13,20 @@ namespace Neo.Ledger
     public class ContractState : ICloneable<ContractState>, ISerializable, IInteroperable
     {
         public int Id;
+        public ushort Version;
```

Comment:
It looks like the developer will add `version` to the extra field, so we'd better rename this `Version`, like `Update`?

Reply:
Done